oak_restricted_kernel_sdk/src/dice.rs

```diff
@@ -56,24 +58,21 @@ fn get_restricted_kernel_dice_data() -> anyhow::Result<RestrictedKernelDiceData>
 }
 
 pub struct Signer {
-    key: SigningKey,
+    key: &'static SigningKey,
 }
 
 impl Signer {
+    pub fn create() -> anyhow::Result<Self> {
+        DICE_WRAPPER
+            .as_ref()
+            .map_err(anyhow::Error::msg)
```

Comment:
DICE_WRAPPER is already a `anyhow::Result`, do you really need this?

Reply:
yes, since DICE_WRAPPER is `anyhow::Result<_, &Error>`. `anyhow::Result<_, Error>` is expected. We could return a static error reference instead I suppose

Reply:
probably best to keep the reallocation though, as otherwise the type signature would lock us into static errors